packages/shared/src/index.ts

```diff
@@ -1,34 +1,28 @@
-/**
- * Errors
+/** The following files are not exported on purpose:
+ * - cookie.ts
```

Comment:
`cookie` imports a third-party package, so to save on bundle-size it's a separate API